operator/cmd/operator/app/operator.go

```diff
@@ -112,7 +112,7 @@ func Run(ctx context.Context, o *options.Options) error {
 		return err
 	}
 
-	// `karmada_operator_build_info` metrics for operator version upgrade
+	// `karmada_build_info` metrics for operator version upgrade
```

Comment:
nice catch!
But could you please add more details about how this helps version upgrade? Or just remove this comment.

Reply:
Thanks @RainbowMango 
Got it